frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -17,16 +17,15 @@ function getSessionReplayLink(): string {
         0
     )
     const link = `https://app.posthog.com/recordings/${posthog?.sessionRecording?.sessionId}?t=${recordingStartTime}`
-    return `\nSession replay: ${link}`
+    return `[Session replay](${link})`
```

Comment:
I like the changes but AFAIK Zendesk doesn't support Markdown, so this wouldn't display correctly…